ext/GlfwOpenGLBackend.jl

```diff
@@ -60,7 +60,8 @@ function renderloop(ui, ctx::Ptr{lib.ImGuiContext}, ::Val{:GlfwOpenGL3};
                     window_size=(1280, 720),
                     window_title="CImGui",
                     engine=nothing,
-                    opengl_version=v"3.2")
+                    opengl_version=v"3.2",
+                    wait_events=false)
```

Comment:
Let's also document it here: https://github.com/JuliaImGui/CImGui.jl/blob/1b338915bbb5e4956916886b1dcc5e1044dd08d2/src/CImGui.jl#L185

And we should mention that it's specific to the GLFW/OpenGL backend (we may add support for more backends in the future). It should have a changelog entry too: https://github.com/JuliaImGui/CImGui.jl/blob/master/docs/src/_changelog.md

If you like you can go ahead and bump the version to 5.1.0 and we can make a new release.